lib/build.gradle.kts

```diff
@@ -101,6 +101,18 @@ tasks.test {
     val poseidonPath = file("${rootDir}/crypto/poseidon/build/bindings").absolutePath
 
     systemProperty("java.library.path", "$libsSharedPath:$pedersenPath:$poseidonPath")
+    systemProperty(
+            "enableIntegrationTests",
+            project.findProperty("enableIntegrationTests")
+                    ?: System.getenv("ENABLE_INTEGRATION_TESTS")
+                    ?: "true",
```

Comment:
I think integration tests should be disabled by default. We should not require any users to set up rpc nodes just to make sure tests passes - by default we should test only with devnet, and make integration tests purely optional.